NDTensors/test/device_list.jl

```diff
@@ -1,3 +1,4 @@
+using GPUArraysCore
```

Comment:
```suggestion
using GPUArraysCore: @allowscalar

```

Reply:
I have added this to the code just haven't made the push to the branch yet. Thanks!

Reply:
Though actually I prefer not having this here and instead putting it at the top of each file that uses `@allowscalar`.

I.e. in test files, put `using Package: f` in files where the file itself actually uses the functionality `f` from `Package`.

Reply:
Okay I will  change to have this at the top of all the files that use `@allowscalar` working on that now so I unresolved this conversation

Reply:
Okay this code is updated! Thanks